src/main/java/com/liferay/faces/util/render/internal/ResourceRendererUtilImpl.java

```diff
@@ -48,9 +48,12 @@
 	// Logger
 	private static final Logger logger = LoggerFactory.getLogger(ResourceRendererUtilImpl.class);
 
+	// Private Constants
+	private static final String RESOURCE_VERIFIER_KEY = ResourceRendererUtilImpl.class.getName() + "_" +
+		"ResourceVerifier";
+
```

Comment:
Although it is not a significant performance impact due to the static nature of this String constant, Shyuang said that for two or three strings, it is better to use String.concat(String) than the plus operator. Let's keep that as a pattern if possible.